packages/esm-patient-search-app/src/patient-search-page/patient-banner/contact-details/contact-details.test.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ContactDetails from './contact-details.component';
+import { useRelationships } from './relationships.resource';
+import { usePatientContactAttributes } from '../hooks/usePatientAttributes';
+
+const mockUseRelationships = useRelationships as jest.Mock;
+const mockUsePatientContactAttributes = usePatientContactAttributes as jest.Mock;
+
+const mockContactAttributes = [{ attributeType: { display: 'Email' }, value: 'test@example.com', uuid: 'email-uuid' }];
+
+const mockRelationships = [
+  {
+    uuid: 'relation-uuid',
```

Comment:
Shall we use an actual uuid? When it comes to mock data let's try to use data that similar to the actual values.

Reply:
done @anjula-sack